package.json

```diff
@@ -8,29 +8,23 @@
   "scripts": {
     "dev": "bun run --watch src/index.ts",
     "start": "bun src/index.ts",
-    "test": "mocha --require ts-node/register tests/**/*.ts",
-    "build:declaration": "tsc --emitDeclarationOnly"
+    "build:declaration": "tsc --emitDeclarationOnly",
+    "tsc": "tsc --noEmit"
   },
   "dependencies": {
     "@sinclair/typebox": "^0.31.17",
-    "@taskforcesh/message-broker": "https://github.com/taskforcesh/message-broker.git#master",
-    "bullmq": "latest",
+    "bullmq": "^5.3.2",
+    "chalk": "^5.3.0",
```

Comment:
any reasons to use chalk? faster options are available.

Reply:
Chalk is well maintained so thats a plus. Usually I pick "pino" though, as a fast and versatile alternative to console.log, maybe it is the best alternative here as well... 🤔

Reply:
+1 on pino, it is awesome. it doesn't work in browsers, but from my understanding you only need it on the server

Reply:
We can change it in a separate PR as it will not imply a breaking change.